### ISSUE TITLE
Mean and std for KITTI

### ISSUE THREAD

**Issue by alwynmathew** [nianticlabs/monodepth2#12]
How did you get [this](https://github.com/nianticlabs/monodepth2/blob/master/networks/resnet_encoder.py#L89) mean and std?
```
x = (input_image - 0.45) / 0.225
```
Why is the input not normalized for PoseCNN network?

https://github.com/nianticlabs/monodepth2/blob/master/networks/pose_cnn.py#L39

**Comment by mrharicot** [comment]
Hi,
This is a rough approximation of the imagenet pretraining which the [pretrained models](https://pytorch.org/docs/stable/torchvision/models.html) expect.

**Comment by mdfirman** [comment]
To be explicit about your PoseCNN question: The PoseCNN doesn't use imagenet pretrained weights, and so we do not transform the inputs.

It is possible that by transforming the inputs for all networks according to the mean and std of the KITTI dataset you might be able to achieve higher scores; we didn't experiment much with this!
